english/eating-walls/find-a-toilet/door/run-in/break-window/coldroom.md

```diff
@@ -0,0 +1,3 @@
+You break the window and pass through it and ended up at a very strange cold room as it is dark but you feel something under neat your feet.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.